client/components/password-change/password-change.js

```diff
@@ -0,0 +1,109 @@
+import "./index.css";
+import React from "react";
+import axios from "axios";
+import qs from "qs";
+import PropTypes from "prop-types";
+
+import {passwordChangeApiUrl} from "../../constants";
+
+import getText from "../../utils/get-text";
+
+// const PasswordChange = ({passwordChange, language, orgSlug}) => {
+export default class PasswordChange extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      newPassword1: "",
+      newPassword2: "",
+    };
+
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(e) {
+    if (e) e.preventDefault();
+    const {orgSlug} = this.props;
+    const url = passwordChangeApiUrl.replace("{orgSlug}", orgSlug);
+    const {newPassword1, newPassword2} = this.state;
```

Comment:
Please add a validation to confirm both the fields have same password before making a request. Something like [this](https://github.com/openwisp/openwisp-wifi-login-pages/blob/226d80e60c3200b4e05a77e7472c1b56c9b90bba/client/components/registration/registration.js#L40)